src/modes/edit_message.cc

```diff
@@ -1045,10 +1045,22 @@ namespace Astroid {
     if (attachments.empty () && !attachment_words.empty ()) {
       ustring bl = body.lowercase ();
 
+      /* strip quoted lines */
+      ostringstream nonquoted;
+      stringstream sstr (bl);
+      while (sstr.good()) {
+        string line;
+        getline (sstr, line);
+
+        if (line[0] != '>')
```

Comment:
Can line be zero length?

Reply:
Are strings not null terminated in C++?

Reply:
[] returns 0 on string length, so is good.